src/lib/seam/connect/seam-http-request.ts

```diff
@@ -128,3 +129,17 @@ export class SeamHttpRequest<
     return this.execute().then(onfulfilled, onrejected)
   }
 }
+
+const getUrlPrefix = (input: string): string => {
+  try {
```

Comment:
can we use `URL.canParse(input)` here rather than try / catch?

Reply:
That's a thing!?! I've wanted that to be a thing for so long 😢

Reply:
ok, it's pretty new, 2023, https://caniuse.com/mdn-api_url_canparse_static